docs/plugins/channel-safely.rst

```diff
@@ -27,6 +27,8 @@ and designations deemed unsafe will be put into :wiki:`Marker Mode <Designations
 Each time a channel designation is completed its group status is checked, and if the group
 is complete pending groups below are made active again.
 
+Settings will persist according to the save game being used.
```

Comment:
I find this wording a bit hard to follow. Autolabor says this: `Once you have enabled it in a fortress, it stays enabled until you explicitly disable it, even if you save and reload your game.` which is more verbose, but also more clear.

Reply:
I find that wording misleading. The plugin doesn't stay enabled, just the features.

Reply:
I could:
```
Features and settings once set will persist until you change them, even if you save and reload your game.
```
Does that work for you?

Reply:
that's good. and plugin enablement will be persisted soon (tm) once I get that working for all plugins.

Reply:
does it have an off switch?

Reply:
```suggestion
Features and settings once set will persist until you change them, even if you save and reload your game.
```